packages/webarchitect/rulesets/base-package.json

```
@@ -0,0 +1,97 @@
+{
+  "$schema": "https://github.com/webarchitect/schemas/main/schema.json",
+  "name": "package",
+  "extends": ["base"],
+  "biome": {
+    "file": {
+      "name": "biome.json",
+      "contains": {
+        "type": "object",
+        "properties": {
+          "$schema": {
+            "type": "string"
+          },
+          "extends": {
+            "type": "array",
+            "items": { "const": "@canonical/biome-config" },
+            "minItems": 1,
+            "maxItems": 1
+          },
+          "files": {
+            "type": "object",
+            "properties": {
+              "include": {
```

```suggestion
              "includes": {
```

Now that we're on Biome v2, the package checks are failing:

```
✗ FAILED VALIDATIONS
──────────────────────────────────────────────────
✗ biome
  Validation failed:
  • files: must NOT have additional properties

✓ PASSED VALIDATIONS
──────────────────────────────────────────────────
✓ package
✓ package-react

Summary: 1/3 validations failed
```